subprojects/docs/src/docs/userguide/depMngmt.adoc

```diff
@@ -544,6 +547,18 @@ Now Gradle will look for your dependencies in the JCenter repository. _jcenter()
             </sample>
 ++++
 
+[[sub:maven_google]]
+==== Maven Google repository
+
+The Google repository hosts Android-specific artifacts including the Android SDK. __Note:__ The contents of the repository cannot be explored from the browser.
```

Comment:
:o: Do we want the Note in there?  Did Google ask us to do it?  That could change at any time, right?